qa-tester/assets/src/js/admin-bar/utils/github.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Base URL for the GitHub API.
+ *
+ * @type {string}
+ */
+const baseUrl = 'https://api.github.com';
```

Comment:
These functions might benefit from error handling. If there's a fetch error, they could send it to the React app, which could then show an error state.

Reply:
Good point. Done in https://github.com/ampproject/amp-wp/pull/4770/commits/099812d19e1954ad2540081ecd3b493e06de87e3.